pyodide-build/pyodide_build/pypabuild.py

```diff
@@ -270,6 +270,7 @@ def get_build_env(
         args["orig__name__"] = __name__
         args["pythoninclude"] = get_build_flag("PYTHONINCLUDE")
         args["PATH"] = env["PATH"]
+        args["F2C_PATH"] = env.get("F2C_PATH", "f2c")
```

Comment:
I believe passing `F2C_PATH` in the cmd line would work without setting it here, then we won't need to modify `pyodide-build`.

Reply:
I tried removing this and it didn't work so I think it's needed. Is there something specific you're suggesting?

Reply:
Hmm, that's strange. Let me check locally.

Reply:
I opened a PR in your fork: https://github.com/hoodmane/pyodide/pull/9. The build passes while tests fail.

Reply:
Looks like you didn't make the tests fail any worse than they already do though. Thanks!